app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] As requested by the previous reviewer, please make sure that your view looks like the wireframe. Currently, it has default divice view:

Your view  | Wireframe view
------------- | -------------
<img width="380" alt="Screenshot 2022-04-21 at 12 19 56" src="https://user-images.githubusercontent.com/77038610/164411817-3ca1bf0d-3e3a-4fe9-9bd1-16a7bf50e365.png"> | <img width="247" alt="Screenshot 2022-04-21 at 12 20 10" src="https://user-images.githubusercontent.com/77038610/164411856-e534c97e-8914-4e8a-8854-ca51e5ba0f74.png"> 

- [ ] Please update login and sign-up views.